docs/components/receive.md

```diff
@@ -147,20 +147,20 @@ Flags:
       --grpc-grace-period=2m     Time to wait after an interrupt received for
                                  GRPC Server.
       --grpc-server-max-connection-age=60m
-                                 The grpc server max connection age. This
-                                 controls how often to re-read the tls
+                                 The grpc server max connection age.
+                                 This controls how often to re-read the tls
```

Comment:
This is the reason why docs check fails I think. We copy this shell output directly here, and there's a mismatch between CI output and your changes. 

I think you might be using some special shell configuration that causes shell outputs to get modified this way, which might also explain why you see other files change when running `make docs`.

Reply:
Can you please elaborate more on what might be the shell config which caused this issue. I use custom config of `zsh+tmux` on an `alacritty` terminal emulator.  Any suggestions on what could be a possible fix

Reply:
So Thanos uses `kingpin` for CLI, which tries to guess your terminal width via a `COLUMNS` env var while printing help/usage text: https://github.com/alecthomas/kingpin/blob/master/guesswidth_unix.go. You can check what it gets set to when you run `make docs` and try to change that to the default 80 value.

If it feels difficult to change, you can just undo the changes done to shell output in the `md` file. 🙂

Reply:
Yeah, that was the issue exactly. my alacritty config had the columns set to 97 which somehow created this fuzz. Wasn't able to exactly correct it through config, manually removed the white-space chnages and hence the CI passed. Thanks